payjoin/src/receive/mod.rs

```diff
@@ -566,6 +567,20 @@ impl PayjoinProposal {
         self.params.disable_output_substitution
     }
 
+    pub fn get_owned_vouts(&self) -> Vec<usize> { self.owned_vouts.clone() }
+
+    pub fn get_payjoin_psbt(&self) -> Psbt { self.payjoin_psbt.clone() }
```

Comment:
is there a good reason to clone here instead of letting `owned_vouts() -> self.owned_vouts` and the same for payjoin_psbt?